traffic_ops/experimental/go-api/tenant/EndPointSeeder.go

```diff
@@ -0,0 +1,202 @@
+        aResponseWriter.WriteHeader(http.StatusNotFound)
+        fmt.Fprint(aResponseWriter, params["id"]+" not found")
+        return
+    }
+    aEndpointSeeder.checkErr(err)
+    
+    var parent_id1 int
+    if parent_id.Valid {
+        parent_id1 = int(parent_id.Int64)
+    } else {
+       parent_id1 = 0        
+    }
+
+    json.NewEncoder(aResponseWriter).Encode(Tenant{Id: id, Name: name, Active: active, ParentId: parent_id1})
+    return
+}
+    
+
+ 
+func (aEndpointSeeder *EndpointSeeder) getListEndpoint(aResponseWriter http.ResponseWriter, aRequest *http.Request) {
+    var tenants []Tenant
+    rows, err := aEndpointSeeder.myDb.Query("SELECT * FROM tenant")
+    aEndpointSeeder.checkErr(err)
+    
+    for rows.Next() {
+        var id int
+        var name string
+        var active bool
+        var parent_id sql.NullInt64
+        var updated time.Time;
+            
+        err = rows.Scan(&id, &name, &active, &parent_id, &updated)
+        aEndpointSeeder.checkErr(err)
+
+        var parent_id1 int
+        if parent_id.Valid {
+            parent_id1 = int(parent_id.Int64)
+        } else {
+            parent_id1 = 0        
+        }
+    
+        tenants = append(tenants, Tenant{Id: id, Name: name, Active: active, ParentId: parent_id1})
+    }
+    
+    json.NewEncoder(aResponseWriter).Encode(tenants)
+
+}
+ 
+func (aEndpointSeeder *EndpointSeeder) createEndpoint(aResponseWriter http.ResponseWriter, aRequest *http.Request) {
+    var tenant Tenant
+    _ = json.NewDecoder(aRequest.Body).Decode(&tenant)
+    
+    var tenantParent sql.NullInt64
+    if tenant.ParentId==0{
+        tenantParent.Valid = false
+    }else{
+    	tenantParent.Int64 = int64(tenant.ParentId)
+    	tenantParent.Valid = true
+    }
+    
+
+    var lastInsertId int
+    err := aEndpointSeeder.myDb.QueryRow("INSERT INTO tenant(name, active, parent_id, last_updated) VALUES($1,$2,$3,$4) returning id;", tenant.Name, tenant.Active, tenantParent, "2012-12-09").Scan(&lastInsertId)
+    aEndpointSeeder.checkErr(err)
+    aEndpointSeeder.getListEndpoint(aResponseWriter, aRequest)
+}
+
+
+
+func (aEndpointSeeder *EndpointSeeder) updateEndpoint(aResponseWriter http.ResponseWriter, aRequest *http.Request) {
+    params := mux.Vars(aRequest)
+    
+    var tenant Tenant
+    _ = json.NewDecoder(aRequest.Body).Decode(&tenant)
+
+    var tenantParent sql.NullInt64
+    if tenant.ParentId==0{
+        tenantParent.Valid = false
+    }else{
+    	tenantParent.Int64 = int64(tenant.ParentId)
+    	tenantParent.Valid = true
+    }
+    
+    stmt, err := aEndpointSeeder.myDb.Prepare("update tenant set name=$1, active=$2, parent_id=$3, last_updated=$4 where id="+params["id"])
+    if err == sql.ErrNoRows{//TODO make it work
+        aResponseWriter.WriteHeader(http.StatusNotFound)
+        return
+    }
+    aEndpointSeeder.checkErr(err)
+    
+    res, err := stmt.Exec(tenant.Name, tenant.Active, tenantParent, "2012-12-09")
```

Comment:
Why 2012? If this is ok, can you pull it into a variable with an explanatory name, e.g. `sampleDataDateStr := "2012-12-09"`?